### DISCUSSION TITLE
🔥🔥 Highly recommend adding these features. 🔥🔥

### DISCUSSION THREAD

**Discussion by ecomdeveloperbd** [inspireui/support#5395] [Working on Features]
# These features are very important in today's world
A most important feature for FluxStore Apps

**1. Popup / Skip ads Or Coupon dialog Popup/ activity ( show some offering products ads when apps opening ) **

Follow demo UI some screenshots
Example: please add these features

Popup offer : 
https://i.imgur.com/zCSAySe.jpg
https://i.imgur.com/dut8Wci.jpg
https://i.imgur.com/45erT8c.jpg

**2. Brand name show multiple rows in the Home screen with click see more options then all bands name show another activity page.**

Follow demo UI some screenshots 
Home Page Brand name show
https://i.imgur.com/p47q4vZ.jpg

All brand names show another activity
https://i.imgur.com/x74NC5d.jpg

**3. Review option add ( Text with image review) features**

Follow demo UI some screenshots
https://i.imgur.com/VJloH8X.jpg

**4. Image Search option. It's a very important feature ( search image  recognition)**

Follow demo UI some screenshots.
https://i.imgur.com/qapDvQZ.jpg
https://i.imgur.com/eFcywVK.jpg
https://i.imgur.com/Z75WUH6.jpg

**5. Gridview or listview configuration store design UI**

Follow demo UI some screenshots
https://i.imgur.com/jIQpg2l.jpg

**And**

**6. Hot Sales or Offering page activity features**

**7. Add image background Multiple product rows**

**Comment by parvej4u2** [comment]
It would be great to add these features to fluxstore.
